### DISCUSSION TITLE
MAUI IMessageBus

### DISCUSSION THREAD

**Discussion by jmichas** [shinyorg/shiny#1158] [Q&A] [answered]
Hi,

I am upgrading a current app to MAUI using the beta Shiny.Framework stuff.
I noticed there is no IMessageBus, is it going away and/or should we be using ReactiveUI.IMessageBus?

Thanks.

**Comment by aritchie** [comment, marked as answer]
Message bus was actually in Shiny.Core as a bunch of libraries were using it, but I've since pulled all of that out.  It seems everyone has a message bus in their offering now, so I removed it from Shiny.
